addons/survey/views/survey_templates.xml

```diff
@@ -178,7 +178,7 @@
                             </p>
                         </div>
                         <a role="button" class="btn btn-primary btn-lg" t-att-href="'/survey/fill/%s/%s' % (survey.access_token, answer.token)">
-                            Start Survey
+                            Start <t t-esc="'Certification' if survey.certificate else 'Survey'" />
```

Comment:
Use a t-if instead. Otherwise the 2 terms will not be translated (I think).

Reply:
Easier to read anyway. Done!